assets/js/learners-bulk-actions.js

```diff
@@ -209,16 +209,11 @@ jQuery( document ).ready( function () {
 			evt.preventDefault();
 			evt.stopPropagation();
 			var $elem = $( this ),
-				$overviewDiv = $elem
-					.siblings( '.learner-course-overview-detail' )
-					.first();
-
-			$learnerCourseOverviewDetail.filter( ':visible' ).slideUp( 'slow' );
-			if ( $overviewDiv.is( ':hidden' ) ) {
-				$overviewDiv.slideDown( 'slow' );
-			} else {
-				$overviewDiv.slideUp( 'slow' );
-			}
+				overviewDiv = $elem.siblings(
```

Comment:
Hmm, isn't `overviewDiv` a jQuery object? Just trying to get why you removed `$` from the name.

Reply:
Oh, I think I got it: it is an array of objects.
In that case, I think we can adjust the variable name to `overviewDivs`?

Reply:
Thanks! Actually having the $ sign for arrays goes with the convention too I think, I've seen it being used for some other places in our code for arrays. And by design there'll only be one div as a sibling in that part of code. I've added the $ sign and updated the PR. Can you kindly take a look again?